pkg/profiler/cpu/cpu.go

```diff
@@ -304,6 +310,19 @@ func (p *CPU) addUnwindTableForProcess(pid int) {
 			level.Debug(p.logger).Log("msg", "failed to add unwind table due to a procfs race", "pid", pid, "err", err)
 		} else if errors.Is(err, errTooManyExecutableMappings) {
 			level.Warn(p.logger).Log("msg", "failed to add unwind table due to having too many executable mappings", "pid", pid, "err", err)
+		} else if errors.Is(err, buildid.ErrTextSectionNotFound) {
+			v, ok := p.processErrorTracker.Peek(err.Error())
```

Comment:
What's the full error string here again? We should make sure it contains the PID and/or mapping file path.

Reply:
I think we should be good here already. The [reported error string](https://github.com/parca-dev/parca-agent/issues/1857#issue-1803351339)(as seen below) already contains pid and mapping start address and mapping file path.
> `level=error name=parca-agent ts=2023-07-13T16:16:20.242517888Z caller=cpu.go:308 component=cpu_profiler msg="failed to add unwind table" pid=1890187 err="setUnwindTableForMapping for
 executable /root/.julia/artifacts/37805a06093bdc0d5938601719cf02fa0cf3aa68/lib/rr/librrpage.so starting at 0x6fffd000 failed: BuildID failed /proc/1890187/root/root/.julia/artifacts/37805a06093bdc0d5938601719cf02fa0cf3aa68/lib/rr/librrpage.so: could not find .text section"`